src/systems/renderer_system.zig

```diff
@@ -186,6 +188,7 @@ fn preUpdate(iter: *ecsu.Iterator(fd.NOCOMP)) void {
     defer ecs.iter_fini(iter.iter);
     const system: *SystemState = @ptrCast(@alignCast(iter.iter.ctx));
     var rctx = system.renderer;
+    rctx.time += iter.iter.delta_time;
```

Comment:
If this is the world time it's probably better to use the one from the Environment singleton (I think it has that?)

Reply:
Yeah:

    const environment_info = ctx.ecsu_world.getSingleton(fd.EnvironmentInfo).?;
    const boosting = state.boost_active_time > environment_info.world_time;

Reply:
Thank you!